pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -545,6 +547,18 @@ func (rc *reconciler) updateDevicePath(volumesNeedUpdate map[v1.UniqueVolumeName
 	} else {
```

Comment:
I noticed that this will do a `kubeClient.CoreV1().Nodes().Get(...)` every time it is called. How frequently will it be called, we should be careful about increase the load on the API server.

Reply:
This is also only called once at most after reconciler starts.  It is when there are volumes need to be updated directly into the actual state which means pod were forcefully deleted during kubelet restarts.

Reply:
Ack.